documentation/source/struct.rst

```diff
@@ -31,6 +31,14 @@ additionally the ``members``, ``protected-members``, ``private-members`` and
 ``undoc-members``
    If specified, the undocumented members of the struct will be displayed.
 
+``membergroups``
+  If specified, only the groups in a comma-delimired list following this
+  directive will be displayed.
+  
+``members-only``
+  This will allow to show only the members. It will remove group names,
+  descriptions as well as the inheritance information.
```

Comment:
Like the class page, can you also update the top paragraph of the heading?

Reply:
will do